### PR TITLE
dotCMS/core#20210 Render markdown content

### PR DESCRIPTION
**Pull request by rjvelazco**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+            },
+            {
+                source: '/dA/:path*',
+                destination: 'https://dotcms.com/dA/:path*'
+            }
+        ];
     }
+    // future: {
+    //     webpack5: true,
+    // },
```
Comment (fmontes) [inline next.config.js]: why?

File: package.json [hunk 2]
```diff
@@ -10,15 +10,21 @@
         "postinstall": "husky install"
     },
     "dependencies": {
+        "@mdx-js/loader": "^1.6.22",
+        "@next/mdx": "^10.1.3",
         "graphql": "^15.5.0",
         "graphql-request": "^3.4.0",
+        "image-size": "^0.9.7",
```
Comment (fmontes) [inline package.json]: We using this?

File: package.json [hunk 3]
```diff
@@ -10,15 +10,21 @@
...
+        "next-mdx-remote": "^2.1.3",
         "npm-check-updates": "^11.3.0",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-is": "^17.0.2",
+        "remark-heading-id": "^1.0.0",
         "styled-components": "^5.2.3",
-        "styled-normalize": "^8.0.7"
+        "styled-normalize": "^8.0.7",
+        "unist-util-visit": "^2.0.3"
```
Comment (fmontes) [inline package.json]: We using this?

File: pages/latest/[urlTitle].tsx [hunk 4]
```diff
@@ -11,9 +13,49 @@ import { DotcmsDocumentation } from '@models/DotcmsDocumentation.model';
...
+import renderToString from 'next-mdx-remote/render-to-string';
+import hydrate from 'next-mdx-remote/hydrate';
+import { MDXProvider } from '@mdx-js/react';
+import { MdxRemote } from 'next-mdx-remote/types';
+import { MDXProviderComponentsProp } from '@mdx-js/react';
+
+const ImageMarkdown = (props) => {
+    const myLoader = ({ src }) => {
+        return `${src}`;
+    };
```
Comment (fmontes) [inline pages/latest/[urlTitle].tsx]: Why do we need this?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: pages/latest/[urlTitle].tsx (hunk 4 above)
Comment (rjvelazco) [reply pages/latest/[urlTitle].tsx]: Yes, we do. If we don't use that function, the src is set using a syntax like: '/_next/image?url=%2FdA%2F73434262-cd90-4d52-b887-5a2f73f3d476%2Fdiagram1&w=1080&q=75' and the image doesn't work.

File: next.config.js (hunk 1 above)
Comment (rjvelazco) [reply next.config.js]: I was getting an error and I thought it was the cause. I forgot to uncomment it.

**Pull request merged**